site/content/_index.html

```diff
@@ -20,13 +20,17 @@
 	<a class="btn btn-lg btn-success mr-3 mb-4" href="https://slack.kopia.io">
 		Developer Discussion <i class="fab fa-slack ml-2 "></i>
 	</a>
-	<p class="font-weight-bold h3 mt-5">Encrypted, Deduplicated, and Compressed Data Backups Using Your Own Cloud Storage</p>
+	<p class="font-weight-bold h3 mt-5">End-to-End Zero-Knowledge Encryption</p>
```

Comment:
i would cut down on that, that's too loud, otherwise looks good.

Reply:
also, where is that background from?

Reply:
We do not have to use the background image. But if we do, we should find some place to credit https://unsplash.com/photos/oyXis2kALVg. The image is free for us to use and we don't need to credit per their license -- but it is the right thing to do.

Hit me up on Slack, I have a few images I am looking at, would be great to have a second opinion on which to use.